src/client/client4.js

```diff
@@ -1095,14 +1302,15 @@ export default class Client4 {
     };
 
     logClientError = async (message, level = 'ERROR') => {
-        const body = {
-            message,
-            level
-        };
+        if (!this.enableLogging) {
```

Comment:
Well not that big of a deal, it won't make it to the server but it will be on the store for when the user reports a problem